c3.js

```diff
@@ -1,7 +1,7 @@
 (function (global, factory) {
-	typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory() :
-	typeof define === 'function' && define.amd ? define(factory) :
-	(global.c3 = factory());
+    typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory() :
```

Comment:
You modified the distribution file!

You should modify the proper file from src directory and not commit the c3.js file which is overwritten during release

Reply:
I think this is just the indentation issue with my editor. I don't see any chages here.

Reply:
What I meant is that you modified the `c3.js` file which is the distribution file and that this file should not be part of the pull request.

Reply:
@panthony i have make the requested changes. I have moved the logic to legend.js and config.js

You can review chnages.